docs/registry/entities/README.md

```diff
@@ -0,0 +1,256 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: Entity Registry
+weight: -2
+auto_gen: below
+--->
+
+<!-- NOTE: THIS FILE IS AUTOGENERATED. DO NOT EDIT BY HAND. -->
+<!-- see templates/registry/markdown/entity_readme.md.j2 -->
+
+# Entity registry
+
+The entities registry is the catalog of available Entities. 
+
+> [!WARNING]
+>
+> The following registry overview is a work in progress.
+>
+> Relationships and signal associations are a work in progress.
+
+Currently, the following namespaces exist:
+<table>
+<thead>
+<tr>
+  <th>Namespace</th>
+  <th>Entity</th>
```

Comment:
maybe 

```suggestion
  <th>Entities</th>
```

?

The rendering is a bit weird

<img width="565" alt="image" src="https://github.com/user-attachments/assets/3f3c69f2-4e23-4db0-92d6-5e532789dc57" />


so it took me  a bit to understand there could be more than one, so maybe plural would be a bit more clear?

Reply:
Yeah - I'm still toying with making this look good. This is a prototype that SHOULD be just enough wrong it give speople a good idea of what it should be :).

Agree it could be called Entities, but if you have other ideas for how to organize the information, I'm all ears.